src/java.desktop/share/classes/java/awt/MenuBar.java

```diff
@@ -342,9 +343,9 @@ final Menu getMenuImpl(int i) {
         Vector<MenuShortcut> shortcuts = new Vector<>();
         int nmenus = getMenuCount();
         for (int i = 0 ; i < nmenus ; i++) {
-            Enumeration<MenuShortcut> e = getMenu(i).shortcuts();
-            while (e.hasMoreElements()) {
-                shortcuts.addElement(e.nextElement());
+            Iterator<MenuShortcut> e = getMenu(i).shortcuts();
+            while (e.hasNext()) {
+                shortcuts.addElement(e.next());
```

Comment:
I think it is fine to replace the Enumeration with the Iterator in most of the places, but here we will get a kind of mix of both, since we cannot remove the usage of Enumeration in the return time.

Reply:
reverted back to Vector here